test/pageobjects/wikiPage.js

```diff
@@ -0,0 +1,48 @@
+class WikiPage {
```

Comment:
You have just one pageObject but in this test you work with 3 pages. Separate this page on 3 pages

Reply:
Resolved